lisa/tools/kdump.py

```diff
@@ -877,8 +877,18 @@ def _check_supported(self, is_auto: bool = False) -> None:
     def _get_resource_disk_dump_path(self) -> str:
         from lisa.features import Disk
 
-        mount_point = self.node.features[Disk].get_resource_disk_mount_point()
-        dump_path = mount_point + "/crash"
+        # Try to access Disk feature (available on Azure platform)
+        try:
+            mount_point = self.node.features[Disk].get_resource_disk_mount_point()
+            dump_path = mount_point + "/crash"
+        except (LisaException, KeyError, AttributeError):
```

Comment:
Which exception are you encountering? It's better to catch the specific one and log it for future troubleshooting.

Reply:
fixed.

Reply:
It's ok to change it to a general "Exception" type. But if you know the exact exception, just use that type. It can help exposing other unknown issue.

Reply:
It looks like a LisaException, Please only catch this type.

Reply:
Yes , done.